mabwiser/utils.py

```diff
@@ -60,3 +62,132 @@ def reset(dictionary: Dict, value) -> NoReturn:
     """
     dictionary.update({}.fromkeys(dictionary, value))
 
+
+class _BaseRNG(metaclass=abc.ABCMeta):
```

Comment:
I think you need to add a seed function and change BaseMAB to call that. For example for default_rng the seeding method is:

seedseq = np.random.SeedSequence(seed)
seeds = seedseq.spawn(total_contexts)

Reply:
Agree that it's a desirable method, one we might need eventually. 

However, currently such functionality is not needed/used anywhere. So I am hesitant to add new features without know proper use cases.

Reply:
Here is what I am referring to for the Generator (default_rng) class: https://numpy.org/devdocs/reference/random/parallel.html#seedsequence-spawn

Reply:
Looking at this again, I'm not sure if we actually need to use these parallel seeding methods. Might be easiest to discuss offline.